test/models/repository_test.rb

```diff
@@ -881,6 +882,12 @@ def test_search_content_view_id
       assert_includes repos, @fedora_17_x86_64
     end
 
+    def test_search_description
+      repos = Repository.search_for("description = \"#{@fedora_17_x86_64.description}\"")
+      assert_includes repos, @fedora_17_x86_64
+      assert_equal repos.length, 1
```

Comment:
I wonder if `assert_equal repos [@fedora_17_x86_64]` could be used instead of 2 asserts.

Reply:
Hey @sean797. I tried that and got a failure. It's because the search_for returns an array to repos. So the comparison doesn't work (correct me if I'm wrong @jturel).